Dockerfile

```diff
@@ -48,9 +33,9 @@ USER appuser
 RUN pip install --upgrade pip \
```

Comment:
this should be removed too, trust the image to provide `pip` otherwise we have two copies in the layers